summit/2018.html

```diff
@@ -57,7 +57,7 @@ <h2 class="text">Schedule</h2>
       <div class="row">
       <div class="col-md-10 col-md-offset-1">
         <img class="sponsor" src="/images/summit/megaphone.svg" width="65px" height="auto" alt="Sponsor"/>
```

Comment:
there were multiple places to change, like the "alt" of the <img>, etc. But fine as a first version.

Reply:
Ah sorry, yeah we are running a bit all over the place this week